Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -323,8 +358,15 @@ Point2D MolDraw2D::getDrawCoords(int at_num) const {
 // ****************************************************************************
 Point2D MolDraw2D::getAtomCoords(const pair<int, int> &screen_cds) const {
   int x = int(double(screen_cds.first) / scale_ + x_min_ - x_trans_);
-  int y = int(double(screen_cds.second) / scale_ + y_min_ - y_trans_);
+  double y =
```

Comment:
This is fairly standard if you want double precision division but need to generate an integer value.  Basically it silences warnings.  rd_cast<int> or static_cast<int> would suffice as well.